### PR TITLE
Homework22: Marie Ambrazhei 

### PR DESCRIPTION
**Pull request by MarieAmbrazhei**

### CODE REVIEW

**Comment by MarieAmbrazhei** [comment]
> Keywords should be stored in the keyword folder, tests should be stored in the tests folder
> 
> https://docs.robotframework.org/docs/examples/project_structure#project-with-tests-resources-and-data-folders
done

File: homework22/robot_tests/test_library.robot [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+Library           homework22/robot_resources/library_keywords.py
+Library           BuiltIn
+Library           OperatingSystem
+
+*** Test Cases ***
+Test Take Book Positive
+    [Documentation]    Test that a user can successfully take a book.
+    ${book}=    Create Book    Learning Python    134527    1200    Mark Lutz
+    ${user}=    Create User    user_id_1
+    ${result}=    Take Book    ${user}    134527
```
Comment (Stml89) [inline homework22/robot_tests/test_library.robot:11]: I think, that it should be like
```suggestion
    ${result}=    Take Book    ${user}    ${book.isbn}
```

Comment (Stml89) [reply homework22/robot_tests/test_library.robot:11]: so, you do not need to repeat the "magic" number many times

Comment (Stml89) [reply homework22/robot_tests/test_library.robot:11]: and in all tests below

**Pull request merged**